go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/mitchellh/copystructure v1.0.0 // indirect
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
+	github.com/prometheus/common v0.4.1
```

Comment:
do we need this?

Reply:
I think, at this point, Goland is just screwing with me by importing a random logging library every time I type in `log.` 😿 Removed